CONTRIBUTING.md

```diff
@@ -91,19 +91,9 @@ bundle exec maze-runner features/handled_errors.feature
 
 #### Pre-release
 
-- [ ] Has all new functionality been manually tested on a release build? Disable development mode in the Unity Build dialog when testing release builds.
-  - [ ] Test that a log message formatted as `SomeTitle: rest of message` generates an error titled `SomeTitle` with message `rest of message`
-  - [ ] Test that a log message formatted without a colon generates an error titled `LogError<level>` with message `rest of message`
-  - [ ] Ensure the example app sends the correct error for each type on iOS
-  - [ ] Ensure OOM reports are sent on iOS when the app is unexpectedly terminated
-  - [ ] Ensure the example app sends the correct error for each type on tvOS
-  - [ ] Ensure the example app sends the correct error for each type on macOS
-  - [ ] Ensure the example app sends the correct error for each type on Android
-  - [ ] Ensure the example app sends the correct error for each type on WebGL
-  - [ ] Ensure the example app sends the correct error for each type on Windows
-  - [ ] Archive the iOS app and validate the bundle type
-  - [ ] Archive the Mac app and validate the bundle type
-  - [ ] Generate a signed APK for Android
+- [ ] Archive the iOS app and validate the bundle type
+- [ ] Archive the Mac app and validate the bundle type
+- [ ] Generate a signed APK for Android
```

Comment:
I don't think this is necessary anymore - a signed APK is generated as part of the E2E tests

Reply:
Agreed, done 👍